testgrid/cmd/configurator/main.go

```diff
@@ -324,11 +324,14 @@ func main() {
 		return
 	}
 
-	// Setup GCS client
+	// Setup GCS client if output is to GCS
```

Comment:
Nit, since you're here: "set up" is the verb, "setup" is a noun. This should be the former.